matrix-sdk-android/src/test/java/org/matrix/android/sdk/api/auth/data/VersionsKtTest.kt

```diff
@@ -62,9 +62,9 @@ class VersionsKtTest {
         Versions(supportedVersions = listOf("r0.6.0")).doesServerSupportThreads() shouldBe false
         Versions(supportedVersions = listOf("r0.9.1")).doesServerSupportThreads() shouldBe false
         Versions(supportedVersions = listOf("v1.2.0")).doesServerSupportThreads() shouldBe false
-        Versions(supportedVersions = listOf("v1.3.0")).doesServerSupportThreads() shouldBe true
-        Versions(supportedVersions = listOf("v1.3.1")).doesServerSupportThreads() shouldBe true
-        Versions(supportedVersions = listOf("v1.5.1")).doesServerSupportThreads() shouldBe true
+        Versions(supportedVersions = listOf("v1.3.0")).doesServerSupportThreads() shouldBe false
+        Versions(supportedVersions = listOf("v1.3.1")).doesServerSupportThreads() shouldBe false
+        Versions(supportedVersions = listOf("v1.5.1")).doesServerSupportThreads() shouldBe false
```

Comment:
Line 71 (so below, but I cannot comment this line) is failing:

<img width="875" alt="image" src="https://user-images.githubusercontent.com/3940906/167711960-bade75a0-2760-4f0d-a3b8-6b5817f87f1b.png">

This is the test
```kotlin
Versions(supportedVersions = listOf("v1.4.0"), unstableFeatures = mapOf("org.matrix.msc3440.stable" to false)).doesServerSupportThreads() shouldBe true
```

and I think that the test has to be updated to `shouldBe false`